README.md

```diff
@@ -100,3 +100,28 @@ response:
   "rtmToken":" " 
 } 
 ```
+
+## Media Encryption Secret ##
+The `secret` endpoint generates a random 32 bytes hex string as secret for Media Encryption.
+
+**endpoint structure**
+```
+/secret
+```
+response:
+``` json
+{"message":"secret in hex"} 
```

Comment:
Let's change this key, the others are "rtmToken" etc.
Maybe "mseSecret/Seed" (media stream encryption), or "encryptionSecret/Seed". What do you think?